src/urllib3/util/ssl_.py

```diff
@@ -320,13 +320,8 @@ def create_urllib3_context(
     # Enable post-handshake authentication for TLS 1.3, see GH #1634. PHA is
     # necessary for conditional client cert authentication with TLS 1.3.
     # The attribute is None for OpenSSL <= 1.1.0 or does not exist in older
-    # versions of Python. We only enable if certificate verification is enabled to work
-    # around Python issue #37428
-    # See: https://bugs.python.org/issue37428
-    if (
-        cert_reqs == ssl.CERT_REQUIRED
-        and getattr(context, "post_handshake_auth", None) is not None
-    ):
+    # versions of Python.
+    if getattr(context, "post_handshake_auth") is not None:
```

Comment:
We might still need a default "None" value in getattr here?

Reply:
My bad, yes it's still needed.